server/pulp/server/db/manage.py

```diff
@@ -54,6 +54,10 @@ def parse_args():
     parser.add_option('--dry-run', action='store_true', dest='dry_run', default=False,
                       help=_('Perform a dry run with no changes made. Returns 1 if there are '
                              'migrations to apply.'))
+
+    parser.add_option('--ignore-running-workers', action='store_true',
+                      dest='ignore_running_workers', default=False,
+                      help=_('Ignore running workers when migration is performed.'))
```

Comment:
I'd like slightly more detailed help text for this, along the lines of "You must ensure that all running workers are stopped before using this flag." I'll be adding more detail to the release notes and announcements.